app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name, health=100, hidden=False):
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.alive.append(self)
```

Comment:
alive is a class attribute so you should reach it from the class and not from the instance of it. same in line 28.

Reply:
Doesn't he inherit it from the Animal class? However, the code works as it should. But I will make changes.

Reply:
the code works correctly because when instance doesn't have alive attribute (and it doesn't) it searches for it in class attribute and then changes it. but it's better to change class attribute directly

Reply:
I understand. Thanks for the explanation!